data/json/professions.json

```
@@ -3423,7 +3423,7 @@
     "type": "profession",
     "ident": "bionic_student",
     "name": "Bionic Student",
-    "description": "Your parents were so obsessed with making sure you aced every test that they had you outfitted with bionics to make you smarter and never forget anything.  And now, you are facing the most dire test yet, and once again you had better succeed, or else.",
+    "description": "Your parents were so obsessed with making sure you aced every test that they had you outfitted with bionics to enhance your intellect and memory.  Now you're facing the hardest test yet.",
```

I think adding a comment on the bionics decreasing in value could be valuable, but don't really care. 
```suggestion
    "description": "Your parents were so obsessed with making sure you aced every test that they had you outfitted with bionics to enhance your intellect and memory.  Now you're facing the hardest test yet, and you're not sure how helpful those outfits are going to be.",
```